VectorsCPP/Vector.hpp

```diff
@@ -0,0 +1,151 @@
+#pragma once
+#include <iostream>
+#include <initializer_list>
+#include <algorithm>
+int capacityNumber = 5;
```

Comment:
is this supposed to be a constant????
i want you to make constants that are calculated at compile time